### DISCUSSION TITLE
Maskphish error

### DISCUSSION THREAD

**Discussion by VladLinux** [jaykali/maskphish#43] [Q&A] [answered]
![20210606_163810](https://user-images.githubusercontent.com/85433319/120918188-af8d1000-c6e5-11eb-9b05-605748b8eff8.jpg)
How to fix this?

**Comment by jaykali** [comment]
We really don't know what is happening. There is no issue from our side. 

Is you have a network connection? Try it again? If doesn't work try removing maskphish and download it again.

We have tried what you have did. See our result, it's working perfectly. Check the screenshot.

![maskphish](https://user-images.githubusercontent.com/62602243/120923376-d6e6db80-c6eb-11eb-97c5-bd2ee6844fab.png)

**Reply by VladLinux** [reply]
I don't know what's happening but i'm sure that my internet connection is very fast.

**Reply by jaykali** [reply]
You told you have a video. If you have the video then please send it.

**Comment by VladLinux** [comment]
I've tried it again but it's just the same even i have strong connection. I forgot to say that maskphish work sometimes and sometimes don't even my net is ok.

**Reply by jaykali** [reply]
You said you have a video. If you have then please send it.

**Reply by VladLinux** [reply]
I have but github won't let me upload it. It says it doesn't support Mp4 format. If you have facebook i can directly send it to you there so you can review the problem.

**Reply by jaykali** [reply]
Facebook blocked us due to an Facebook hacking article. You can send it us by various methods uploading it on other services. Or we are in Twitter and Telegram. https://t.me/kalilinuxin

**Comment by jaykali** [comment, marked as answer]
Use VPN. Your ISP is blocking our services. Thanks.
